src/main/java/Task.java

```diff
@@ -0,0 +1,68 @@
+public class Task{
```

Comment:
Perhaps a space after the class name? I noticed the same issue in several other places too.
```suggestion
public class Task {
```